build.gradle

```diff
@@ -61,6 +61,11 @@ jar {
     baseName = 'gds-metrics-dropwizard'
 }
 
+def dependencyVersions = [
+        dropwizard: '1.3.5',
+        mockito: '2.23.0',
```

Comment:
why did you extract a variable for `mockito` here when it's only used once?

Reply:
when I was trying to get this working on 11 I added powermock there as well, but then had to completely remove powermock, which left mockito